ServerCore/Startup.cs

```diff
@@ -149,6 +150,13 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<ServerMessageListener>();
             services.AddSingleton<PresenceStore>();
             services.AddSingleton<NotificationHelper>();
+
+            // Require emails to be unique so that we can be resilient to external provider login changes.
+            // Search the codebase for "RequireUniqueEmail" for the code that requires this.
+            services.Configure<IdentityOptions>(options =>
+            {
+                options.User.RequireUniqueEmail = true;
```

Comment:
Note that I have not verified that emails are actually unique at present; I would not be surprised if some staffers have some duplication generated during server development, or perhaps some migrations have left stale users behind. Verification/cleanup is required before this code can be merged.

Reply:
I've had to delete the existing AspNet user in order to fix this issue in the past, so I suspect there are few to no duplicates.